packages/blade/src/components/Table/TablePagination.web.tsx

```diff
@@ -242,7 +243,7 @@ const _TablePagination = ({
       display="flex"
       flexDirection="row"
       padding={tablePagination.padding}
-      backgroundColor={tablePagination.backgroundColor}
+      backgroundColor={backgroundColor}
```

Comment:
Take the backgroundColor from `useTableContext` Similar to how we have done it on TableToolbar.web.tsx

Reply:
Thanks @chaitanyadeorukhkar 🙌, Done.